app/backend/approaches/retrievethenread.py

```diff
@@ -118,7 +118,7 @@ async def run(
                 # Azure Open AI takes the deployment name as the model name
                 model=self.chatgpt_deployment if self.chatgpt_deployment else self.chatgpt_model,
                 messages=message_builder.messages,
-                temperature=overrides.get("temperature") or 0.3,
```

Comment:
Interesting that our default is 0.3 for RTR, didnt realize thst.

Reply:
I think these choices were made a long time ago and never revisited. Perhaps we can use the evaluation repo on the samples to figure out which one is the best :)

Reply:
Yep good idea! I've documented for now.